gathering/async/get.js

```diff
@@ -56,23 +56,30 @@ function reduceUpdates (thread) {
 }
 
 function reduceAttendees (myKey, thread) {
-  const attendees = thread
+  const [attendees, unAttendees] = thread
     .filter(isAttendee)
-    .reduce((acc, msg) => {
+    .reduce(([accAttendee, accUnattendee], msg) => {
       const { link, remove } = msg.value.content.attendee
       // only trust attendee calls from people themselves for now
       if (msg.value.author !== link) return acc
```

Comment:
oh, my linted also caught this - `acc not defined`
I've added standard to the `npm test` script.

I'm using vim and find [ALE](https://github.com/w0rp/ale) to be excellent and save me a lot time over little bugs like this. Highly recommend having something in place